### ISSUE TITLE
% Search shortcut on location bar is limited to container

### ISSUE THREAD

**Issue by robsonsobral** [mozilla/multi-account-containers#2387]
### Before submitting a bug report

- [X] I updated to the latest version of Multi-Account Container and tested if I can reproduce the issue
- [X] I searched for existing reports to see if it hasn't already been reported

### Step to reproduce

1. type `%` on location bar;
2. type something from title or URL from an opened tabs on a different container;

### Actual behavior

The opened tab isn't found.

### Expected behavior

The limitation of the search within the used container defies the purpose of the search shortcut

### Provide a copy of Troubleshooting Information page

```plain text
Application Basics
------------------

Name: Firefox
Version: 103.0b5
Build ID: 20220705185846
Distribution ID:
Update Channel: aurora
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:103.0) Gecko/20100101 Firefox/103.0
OS: Windows_NT 10.0 22000
Launcher Process: Enabled
Multiprocess Windows: 36/36
Fission Windows: 36/36 Enabled by default
Remote Processes: 129
Enterprise Policies: Inactive
Google Location Service Key: Found
Google Safebrowsing Key: Found
Mozilla Location Service Key: Found
Safe Mode: false
Memory Size (RAM): 31.9 GB
Disk Space Available: 236 GB

Crash Reports for the Last 3 Days
---------------------------------

Firefox Developer Edition Features
----------------------------------

Name: DoH Roll-Out
Version: 2.0.0
ID: doh-rollout@mozilla.org

Name: Firefox Screenshots
Version: 39.0.1
ID: screenshots@mozilla.org

Name: Form Autofill
Version: 1.0.1
ID: formautofill@mozilla.org

Name: Picture-In-Picture
Version: 1.0.0
ID: pictureinpicture@mozilla.org

Name: Web Compatibility Interventions
Version: 103.0.0
ID: webcompat@mozilla.org

Name: WebCompat Reporter
Version: 1.5.0
ID: webcompat-reporter@mozilla.org

Remote Features
---------------

Remote Processes
----------------

Type: Web Content
Count: 1 / 8

Type: Privileged About
Count: 1

Type: Extension
Count: 1

Type: Isolated Web Content
Count: 119

Type: Isolated Service Worker
Count: 5

Type: GPU
Count: 1

Type: Socket
Count: 1

Add-ons
-------

Name: English United States Dictionary
Type: dictionary
Version: 91.0
Enabled: true
ID: @unitedstatesenglishdictionary

Name: Verificador Ortográfico para Português do Brasil
Type: dictionary
Version: 2.5-3.2.13.1webext
Enabled: true
ID: pt-BR@dellalibera.sf.net

Name: Add-ons Search Detection
Type: extension
Version: 2.0.0
Enabled: true
ID: addons-search-detection@mozilla.com

Name: Always on Top
Type: extension
Version: 6.3
Enabled: true
ID: {E6C93316-271E-4b3d-8D7E-FE11B4350AEB}

Name: Amazon.co.uk
Type: extension
Version: 1.9
Enabled: true
ID: amazon@search.mozilla.org

Name: Amazon.com
Type: extension
Version: 1.3
Enabled: true
ID: amazondotcom@search.mozilla.org

Name: axe DevTools - Web Accessibility Testing
Type: extension
Version: 4.31.2
Enabled: true
ID: @axe-firefox-devtools

Name: Bing
Type: extension
Version: 1.3
Enabled: true
ID: bing@search.mozilla.org

Name: burlesco
Type: extension
Version: 12.2
Enabled: true
ID: burlesco@burles.co

Name: Context Search
Type: extension
Version: 4.3.0
Enabled: true
ID: olivier.debroqueville@gmail.com

Name: Decentraleyes
Type: extension
Version: 2.0.17
Enabled: true
ID: jid1-BoFifL9Vbdl2zQ@jetpack

Name: DuckDuckGo
Type: extension
Version: 1.1
Enabled: true
ID: ddg@search.mozilla.org

Name: Duplicate Tabs Closer
Type: extension
Version: 3.5.3
Enabled: true
ID: jid0-RvYT2rGWfM8q5yWxIxAHYAeo5Qg@jetpack

Name: eBay
Type: extension
Version: 1.3
Enabled: true
ID: ebay@search.mozilla.org

Name: Enhancer for YouTube™
Type: extension
Version: 2.0.115
Enabled: true
ID: enhancerforyoutube@maximerf.addons.mozilla.org

Name: Facebook Container
Type: extension
Version: 2.3.2
Enabled: true
ID: @contain-facebook

Name: Feedly Mini
Type: extension
Version: 1.0.10
Enabled: true
ID: appstore-mini@feedly.com

Name: Firefox Multi-Account Containers
Type: extension
Version: 8.0.7
Enabled: true
ID: @testpilot-containers

Name: Form History Control (II)
Type: extension
Version: 2.5.6.1
Enabled: true
ID: formhistory@yahoo.com

Name: FoxyTab
Type: extension
Version: 2.29
Enabled: true
ID: foxytab@eros.man

Name: Ghostery – Privacy Ad Blocker
Type: extension
Version: 8.7.4
Enabled: true
ID: firefox@ghostery.com

Name: Google
Type: extension
Version: 1.2
Enabled: true
ID: google@search.mozilla.org

Name: Grammar & Spell Checker — LanguageTool
Type: extension
Version: 5.3.0
Enabled: true
ID: languagetool-webextension@languagetool.org

Name: KeePassXC-Browser
Type: extension
Version: 1.8.0
Enabled: true
ID: keepassxc-browser@keepassxc.org

Name: Lighthouse
Type: extension
Version: 100.0.0.2
Enabled: true
ID: {cf3dba12-a848-4f68-8e2d-f9fadc0721de}

Name: Markdown Here
Type: extension
Version: 2.13.1
Enabled: true
ID: markdown-here-webext@adam.pritchard

Name: Panorama View
Type: extension
Version: 0.9.3
Enabled: true
ID: {60e27487-c779-464c-8698-ad481b718d5f}

Name: Privacy Pass
Type: extension
Version: 3.0.3
Enabled: true
ID: {48748554-4c01-49e8-94af-79662bf34d50}

Name: Sidebery
Type: extension
Version: 4.10.1
Enabled: true
ID: {3c078156-979c-498b-8990-85f7987dd929}

Name: Tab Center Reborn
Type: extension
Version: 2.3.1
Enabled: true
ID: tabcenter-reborn@ariasuni

Name: Tab Manager Plus for Firefox
Type: extension
Version: 5.2.0
Enabled: true
ID: {45f2dc53-96cd-4c41-91f6-f4a73a8fb2b0}

Name: Tab Session Manager
Type: extension
Version: 6.12.0
Enabled: true
ID: Tab-Session-Manager@sienori

Name: Toggl Track: Productivity & Time Tracker
Type: extension
Version: 2.0.15
Enabled: true
ID: toggl-button@toggl.com

Name: Vue.js devtools
Type: extension
Version: 6.2.0
Enabled: true
ID: {5caff8cc-3d2e-4110-a88a-003cc85b3858}

Name: Web Developer Checklist
Type: extension
Version: 2.0.3
Enabled: true
ID: {65bf4dd0-b6df-44f7-9285-cb5545037d80}

Name: Wikipedia (en)
Type: extension
Version: 1.1
Enabled: true
ID: wikipedia@search.mozilla.org

Name: Window Merger
Type: extension
Version: 0.6.0
Enabled: true
ID: {a116b6c7-0611-4938-ba3d-2e4c61241128}

Name: AdNauseam
Type: extension
Version: 3.13.1
Enabled: false
ID: adnauseam@rednoise.org

Name: All Tabs Helper
Type: extension
Version: 1.2.45
Enabled: false
ID: alltabshelper@alltabshelper.org

Name: app.telemetry Page Speed Monitor
Type: extension
Version: 17.1.10
Enabled: false
ID: jid0-OqxcSY9VMeMm8jJqJYy5KM0nmS8@jetpack

Name: Auto Tab Discard
Type: extension
Version: 0.5.0
Enabled: false
ID: {c2c003ee-bd69-42a2-b0e9-6f34222cb046}

Name: ClearURLs
Type: extension
Version: 1.24.1
Enabled: false
ID: {74145f27-f039-47ce-a470-a662b129930a}

Name: Close n forget
Type: extension
Version: 0.11.6.1-signed.1-signed
Enabled: false
ID: closeforget@addons.mozilla.org

Name: Cookie AutoDelete
Type: extension
Version: 3.8.1
Enabled: false
ID: CookieAutoDelete@kennydo.com

Name: CORS Everywhere
Type: extension
Version: 18.11.13.2043
Enabled: false
ID: cors-everywhere@spenibus

Name: CSS Stacking Context inspector
Type: extension
Version: 1.1.11
Enabled: false
ID: {239a3710-9399-4d6e-a7ba-de533a5177fb}

Name: DevTools z-index
Type: extension
Version: 1.1.0
Enabled: false
ID: {40c9a151-fca9-4252-aa05-cb0df43d18b7}

Name: Disconnect
Type: extension
Version: 20.3.1.1
Enabled: false
ID: 2.0@disconnect.me

Name: DuckDuckGo Privacy Essentials
Type: extension
Version: 2022.6.27
Enabled: false
ID: jid1-ZAdIEUB7XOzOJw@jetpack

Name: Feature Queries Manager
Type: extension
Version: 0.3
Enabled: false
ID: {18f7c9a9-5370-4060-9e77-07cb76c28f5e}

Name: Feedly Subscribe Button
Type: extension
Version: 0.9.0
Enabled: false
ID: {a8ffd15d-67d0-427f-80ad-07a7d76431e1}

Name: Find & Replace for Text Editing
Type: extension
Version: 1.3.17
Enabled: false
ID: {ad1a1412-512a-4955-8bea-3ac63a5d970b}

Name: Firefox DevTools ADB Extension
Type: extension
Version: 0.0.5
Enabled: false
ID: adb@mozilla.org

Name: HTTP Version Indicator
Type: extension
Version: 3.2.1
Enabled: false
ID: spdyindicator@chengsun.github.com

Name: HTTPS Everywhere
Type: extension
Version: 2021.7.13
Enabled: false
ID: https-everywhere@eff.org

Name: I don't care about cookies
Type: extension
Version: 3.4.1
Enabled: false
ID: jid1-KKzOGWgsW3Ao4Q@jetpack

Name: Iridium for YouTube
Type: extension
Version: 1.2.7
Enabled: false
ID: unlisted-iridium@particlecore.github.io

Name: JavaScript Toggle On and Off
Type: extension
Version: 0.2.4
Enabled: false
ID: {479f0278-2c34-4365-b9f0-1d328d0f0a40}

Name: Laboratory
Type: extension
Version: 3.0.8
Enabled: false
ID: 1b2383b324c8520974ee097e46301d5ca4e076de387c02886f1c6b1503671586@pokeinthe.io

Name: Mailvelope
Type: extension
Version: 4.6.0
Enabled: false
ID: jid1-AQqSMBYb0a8ADg@jetpack

Name: Max Tabs (Web Extension)
Type: extension
Version: 0.1.0
Enabled: false
ID: max-tabs@kanru

Name: MessPostage
Type: extension
Version: 0.0.7
Enabled: false
ID: {cd982b7a-ece2-4476-a2bc-5b0d085ccbcd}

Name: Microsoft Power Automate
Type: extension
Version: 2.0.11
Enabled: false
ID: pad.firefox@microsoft.com

Name: Neat URL
Type: extension
Version: 5.0.0
Enabled: false
ID: neaturl@hugsmile.eu

Name: netlify-browser-extension
Type: extension
Version: 1.2.2
Enabled: false
ID: {bd3e8d0d-2198-4b65-9162-a2e889ca19fd}

Name: New Tab Override
Type: extension
Version: 15.1.1
Enabled: false
ID: newtaboverride@agenedia.com

Name: ng-inspect
Type: extension
Version: 1.0
Enabled: false
ID: jid1-v6jvJrqACQCakw@jetpack

Name: Notes by Firefox
Type: extension
Version: 4.3.7
Enabled: false
ID: notes@mozilla.com

Name: PageSpeed Insights
Type: extension
Version: 0.1.0.0
Enabled: false
ID: {f9263e7a-8e0a-4017-a394-dc1ae3b1252d}

Name: Pocket Sidebar
Type: extension
Version: 1.0.0
Enabled: false
ID: {64690374-6b5e-4948-90dd-336ed1ac1b6b}

Name: Privacy Badger
Type: extension
Version: 2021.11.23.1
Enabled: false
ID: jid1-MnnxcxisBPnSXQ@jetpack

Name: Script Blocker Ultimate
Type: extension
Version: 4.1.1
Enabled: false
ID: {660EBB8E-4E21-11E9-9A47-13E64A802C61}

Name: Search By Image (by Google)
Type: extension
Version: 1.1.4
Enabled: false
ID: {ce7e73df-6a44-4028-8079-5927a588c948}

Name: Session Manager
Type: extension
Version: 0.8.1.13
Enabled: false
ID: {1280606b-2510-4fe0-97ef-9b5a22eafe30}

Name: Simple Tab Groups
Type: extension
Version: 4.7.2.1
Enabled: false
ID: simple-tab-groups@drive4ik

Name: snoozetabs
Type: extension
Version: 1.0.20
Enabled: false
ID: snoozetabs@mozilla.com

Name: Stylus
Type: extension
Version: 1.5.25
Enabled: false
ID: {7a7a4a92-a2a0-41d1-9fd7-1e92480d612d}

Name: Tab Center Redux
Type: extension
Version: 0.7.1
Enabled: false
ID: {0ad88674-2b41-4cfb-99e3-e206c74a0076}

Name: Tab Stack for Firefox
Type: extension
Version: 1.0.0
Enabled: false
ID: tabstack@github

Name: Tab Workspaces
Type: extension
Version: 1.2.0
Enabled: false
ID: {aba5dba6-dc7b-458d-8587-d8686f815531}

Name: Template Extractor
Type: extension
Version: 1.8.1
Enabled: false
ID: dinsa@dsic.upv.es

Name: TinEye Reverse Image Search
Type: extension
Version: 1.5.2
Enabled: false
ID: tineye@ideeinc.com

Name: Tree Style Tab
Type: extension
Version: 3.8.25
Enabled: false
ID: treestyletab@piro.sakura.ne.jp

Name: TST More Tree Commands
Type: extension
Version: 1.4
Enabled: false
ID: tst-more-tree-commands@piro.sakura.ne.jp

Name: Unload Tab
Type: extension
Version: 1.2
Enabled: false
ID: {b3fdb2a9-f31a-4f12-b7b3-085aba679868}

Name: Wappalyzer - Technology profiler
Type: extension
Version: 6.10.36
Enabled: false
ID: wappalyzer@crunchlabz.com

Name: WAVE Evaluation Tool
Type: extension
Version: 3.1.6
Enabled: false
ID: {9bbf6724-d709-492e-a313-bfed0415a224}

Name: Wayback Machine
Type: extension
Version: 3.1
Enabled: false
ID: wayback_machine@mozilla.org

Name: Web Developer
Type: extension
Version: 2.0.5
Enabled: false
ID: {c45c406e-ab73-11d8-be73-000a95be3b12}

Name: z-index visualizer
Type: extension
Version: 1.0
Enabled: false
ID: {db18b2b9-896c-49c9-a64f-73ce52b80ee5}

Security Software
-----------------

Type: Microsoft Defender Antivirus

Type:

Type: Microsoft Firewall

Graphics
--------

Features
Compositing: WebRender
Asynchronous Pan/Zoom: wheel input enabled; scrollbar drag enabled; keyboard enabled; autoscroll enabled; smooth pinch-zoom enabled
WebGL 1 Driver WSI Info: EGL_VENDOR: Google Inc. (NVIDIA) EGL_VERSION: 1.5 (ANGLE 2.1.15727 git hash: 23851a53779d) EGL_EXTENSIONS: EGL_EXT_create_context_robustness EGL_ANGLE_d3d_share_handle_client_buffer EGL_ANGLE_d3d_texture_client_buffer EGL_ANGLE_surface_d3d_texture_2d_share_handle EGL_ANGLE_query_surface_pointer EGL_ANGLE_window_fixed_size EGL_ANGLE_keyed_mutex EGL_ANGLE_surface_orientation EGL_ANGLE_direct_composition EGL_NV_post_sub_buffer EGL_KHR_create_context EGL_KHR_image EGL_KHR_image_base EGL_KHR_gl_texture_2D_image EGL_KHR_gl_texture_cubemap_image EGL_KHR_gl_renderbuffer_image EGL_KHR_get_all_proc_addresses EGL_KHR_stream EGL_KHR_stream_consumer_gltexture EGL_NV_stream_consumer_gltexture_yuv EGL_ANGLE_flexible_surface_compatibility EGL_ANGLE_stream_producer_d3d_texture EGL_ANGLE_create_context_webgl_compatibility EGL_CHROMIUM_create_context_bind_generates_resource EGL_CHROMIUM_sync_control EGL_EXT_pixel_format_float EGL_KHR_surfaceless_context EGL_ANGLE_display_texture_share_group EGL_ANGLE_display_semaphore_share_group EGL_ANGLE_create_context_client_arrays EGL_ANGLE_program_cache_control EGL_ANGLE_robust_resource_initialization EGL_ANGLE_create_context_extensions_enabled EGL_ANDROID_blob_cache EGL_ANDROID_recordable EGL_ANGLE_image_d3d11_texture EGL_ANGLE_create_context_backwards_compatible EGL_KHR_create_context_no_error EGL_KHR_reusable_sync EGL_EXTENSIONS(nullptr): EGL_EXT_client_extensions EGL_EXT_device_query EGL_EXT_platform_base EGL_EXT_platform_device EGL_ANGLE_platform_angle EGL_ANGLE_platform_angle_d3d EGL_ANGLE_platform_angle_d3d11on12 EGL_ANGLE_device_creation EGL_ANGLE_device_creation_d3d11 EGL_ANGLE_experimental_present_path EGL_KHR_client_get_all_proc_addresses EGL_KHR_debug EGL_ANGLE_explicit_context EGL_ANGLE_feature_control IsWebglOutOfProcessEnabled: 1
WebGL 1 Driver Renderer: Google Inc. (NVIDIA) -- ANGLE (NVIDIA, NVIDIA GeForce RTX 2060 Direct3D11 vs_5_0 ps_5_0, D3D11-30.0.15.1295)
WebGL 1 Driver Version: OpenGL ES 2.0.0 (ANGLE 2.1.15727 git hash: 23851a53779d)
WebGL 1 Driver Extensions: GL_ANGLE_base_vertex_base_instance GL_ANGLE_client_arrays GL_ANGLE_depth_texture GL_ANGLE_explicit_context GL_ANGLE_explicit_context_gles1 GL_ANGLE_framebuffer_blit GL_ANGLE_framebuffer_multisample GL_ANGLE_get_serialized_context_string GL_ANGLE_get_tex_level_parameter GL_ANGLE_instanced_arrays GL_ANGLE_lossy_etc_decode GL_ANGLE_memory_size GL_ANGLE_multi_draw GL_ANGLE_multiview_multisample GL_ANGLE_pack_reverse_row_order GL_ANGLE_program_cache_control GL_ANGLE_provoking_vertex GL_ANGLE_request_extension GL_ANGLE_texture_compression_dxt3 GL_ANGLE_texture_compression_dxt5 GL_ANGLE_texture_usage GL_ANGLE_translated_shader_source GL_CHROMIUM_bind_generates_resource GL_CHROMIUM_bind_uniform_location GL_CHROMIUM_color_buffer_float_rgb GL_CHROMIUM_color_buffer_float_rgba GL_CHROMIUM_copy_compressed_texture GL_CHROMIUM_copy_texture GL_CHROMIUM_lose_context GL_CHROMIUM_sync_query GL_EXT_EGL_image_external_wrap_modes GL_EXT_blend_func_extended GL_EXT_blend_minmax GL_EXT_color_buffer_half_float GL_EXT_debug_label GL_EXT_debug_marker GL_EXT_discard_framebuffer GL_EXT_disjoint_timer_query GL_EXT_draw_buffers GL_EXT_draw_elements_base_vertex GL_EXT_float_blend GL_EXT_frag_depth GL_EXT_instanced_arrays GL_EXT_map_buffer_range GL_EXT_multisampled_render_to_texture GL_EXT_occlusion_query_boolean GL_EXT_read_format_bgra GL_EXT_robustness GL_EXT_sRGB GL_EXT_shader_texture_lod GL_EXT_texture_compression_bptc GL_EXT_texture_compression_dxt1 GL_EXT_texture_compression_rgtc GL_EXT_texture_compression_s3tc_srgb GL_EXT_texture_filter_anisotropic GL_EXT_texture_format_BGRA8888 GL_EXT_texture_rg GL_EXT_texture_storage GL_EXT_unpack_subimage GL_KHR_debug GL_KHR_no_error GL_KHR_parallel_shader_compile GL_KHR_robust_buffer_access_behavior GL_NV_EGL_stream_consumer_external GL_NV_fence GL_NV_pack_subimage GL_NV_pixel_buffer_object GL_OES_EGL_image GL_OES_EGL_image_external GL_OES_compressed_EAC_R11_signed_texture GL_OES_compressed_EAC_R11_unsigned_texture GL_OES_compressed_EAC_RG11_signed_texture GL_OES_compressed_EAC_RG11_unsigned_texture GL_OES_compressed_ETC2_RGB8_texture GL_OES_compressed_ETC2_RGBA8_texture GL_OES_compressed_ETC2_punchthroughA_RGBA8_texture GL_OES_compressed_ETC2_punchthroughA_sRGB8_alpha_texture GL_OES_compressed_ETC2_sRGB8_alpha8_texture GL_OES_compressed_ETC2_sRGB8_texture GL_OES_depth24 GL_OES_depth32 GL_OES_draw_elements_base_vertex GL_OES_element_index_uint GL_OES_fbo_render_mipmap GL_OES_get_program_binary GL_OES_mapbuffer GL_OES_packed_depth_stencil GL_OES_rgb8_rgba8 GL_OES_standard_derivatives GL_OES_surfaceless_context GL_OES_texture_border_clamp GL_OES_texture_float GL_OES_texture_float_linear GL_OES_texture_half_float GL_OES_texture_half_float_linear GL_OES_texture_npot GL_OES_texture_stencil8 GL_OES_vertex_array_object GL_WEBGL_video_texture
WebGL 1 Extensions: ANGLE_instanced_arrays EXT_blend_minmax EXT_color_buffer_half_float EXT_float_blend EXT_frag_depth EXT_shader_texture_lod EXT_sRGB EXT_texture_compression_bptc EXT_texture_compression_rgtc EXT_texture_filter_anisotropic MOZ_debug OES_element_index_uint OES_fbo_render_mipmap OES_standard_derivatives OES_texture_float OES_texture_float_linear OES_texture_half_float OES_texture_half_float_linear OES_vertex_array_object WEBGL_color_buffer_float WEBGL_compressed_texture_s3tc WEBGL_compressed_texture_s3tc_srgb WEBGL_debug_renderer_info WEBGL_debug_shaders WEBGL_depth_texture WEBGL_draw_buffers WEBGL_lose_context
WebGL 2 Driver WSI Info: EGL_VENDOR: Google Inc. (NVIDIA) EGL_VERSION: 1.5 (ANGLE 2.1.15727 git hash: 23851a53779d) EGL_EXTENSIONS: EGL_EXT_create_context_robustness EGL_ANGLE_d3d_share_handle_client_buffer EGL_ANGLE_d3d_texture_client_buffer EGL_ANGLE_surface_d3d_texture_2d_share_handle EGL_ANGLE_query_surface_pointer EGL_ANGLE_window_fixed_size EGL_ANGLE_keyed_mutex EGL_ANGLE_surface_orientation EGL_ANGLE_direct_composition EGL_NV_post_sub_buffer EGL_KHR_create_context EGL_KHR_image EGL_KHR_image_base EGL_KHR_gl_texture_2D_image EGL_KHR_gl_texture_cubemap_image EGL_KHR_gl_renderbuffer_image EGL_KHR_get_all_proc_addresses EGL_KHR_stream EGL_KHR_stream_consumer_gltexture EGL_NV_stream_consumer_gltexture_yuv EGL_ANGLE_flexible_surface_compatibility EGL_ANGLE_stream_producer_d3d_texture EGL_ANGLE_create_context_webgl_compatibility EGL_CHROMIUM_create_context_bind_generates_resource EGL_CHROMIUM_sync_control EGL_EXT_pixel_format_float EGL_KHR_surfaceless_context EGL_ANGLE_display_texture_share_group EGL_ANGLE_display_semaphore_share_group EGL_ANGLE_create_context_client_arrays EGL_ANGLE_program_cache_control EGL_ANGLE_robust_resource_initialization EGL_ANGLE_create_context_extensions_enabled EGL_ANDROID_blob_cache EGL_ANDROID_recordable EGL_ANGLE_image_d3d11_texture EGL_ANGLE_create_context_backwards_compatible EGL_KHR_create_context_no_error EGL_KHR_reusable_sync EGL_EXTENSIONS(nullptr): EGL_EXT_client_extensions EGL_EXT_device_query EGL_EXT_platform_base EGL_EXT_platform_device EGL_ANGLE_platform_angle EGL_ANGLE_platform_angle_d3d EGL_ANGLE_platform_angle_d3d11on12 EGL_ANGLE_device_creation EGL_ANGLE_device_creation_d3d11 EGL_ANGLE_experimental_present_path EGL_KHR_client_get_all_proc_addresses EGL_KHR_debug EGL_ANGLE_explicit_context EGL_ANGLE_feature_control IsWebglOutOfProcessEnabled: 1
WebGL 2 Driver Renderer: Google Inc. (NVIDIA) -- ANGLE (NVIDIA, NVIDIA GeForce RTX 2060 Direct3D11 vs_5_0 ps_5_0, D3D11-30.0.15.1295)
WebGL 2 Driver Version: OpenGL ES 3.0.0 (ANGLE 2.1.15727 git hash: 23851a53779d)
WebGL 2 Driver Extensions: GL_ANGLE_base_vertex_base_instance GL_ANGLE_client_arrays GL_ANGLE_copy_texture_3d GL_ANGLE_depth_texture GL_ANGLE_explicit_context GL_ANGLE_explicit_context_gles1 GL_ANGLE_framebuffer_blit GL_ANGLE_framebuffer_multisample GL_ANGLE_get_serialized_context_string GL_ANGLE_get_tex_level_parameter GL_ANGLE_instanced_arrays GL_ANGLE_lossy_etc_decode GL_ANGLE_memory_size GL_ANGLE_multi_draw GL_ANGLE_multiview_multisample GL_ANGLE_pack_reverse_row_order GL_ANGLE_program_cache_control GL_ANGLE_provoking_vertex GL_ANGLE_request_extension GL_ANGLE_texture_compression_dxt3 GL_ANGLE_texture_compression_dxt5 GL_ANGLE_texture_multisample GL_ANGLE_texture_usage GL_ANGLE_translated_shader_source GL_CHROMIUM_bind_generates_resource GL_CHROMIUM_bind_uniform_location GL_CHROMIUM_copy_compressed_texture GL_CHROMIUM_copy_texture GL_CHROMIUM_lose_context GL_CHROMIUM_sync_query GL_EXT_EGL_image_external_wrap_modes GL_EXT_blend_func_extended GL_EXT_blend_minmax GL_EXT_color_buffer_float GL_EXT_color_buffer_half_float GL_EXT_debug_label GL_EXT_debug_marker GL_EXT_discard_framebuffer GL_EXT_disjoint_timer_query GL_EXT_draw_buffers GL_EXT_draw_buffers_indexed GL_EXT_draw_elements_base_vertex GL_EXT_float_blend GL_EXT_frag_depth GL_EXT_instanced_arrays GL_EXT_map_buffer_range GL_EXT_multisampled_render_to_texture GL_EXT_occlusion_query_boolean GL_EXT_read_format_bgra GL_EXT_robustness GL_EXT_sRGB GL_EXT_shader_texture_lod GL_EXT_texture_compression_bptc GL_EXT_texture_compression_dxt1 GL_EXT_texture_compression_rgtc GL_EXT_texture_compression_s3tc_srgb GL_EXT_texture_filter_anisotropic GL_EXT_texture_format_BGRA8888 GL_EXT_texture_norm16 GL_EXT_texture_rg GL_EXT_texture_storage GL_EXT_unpack_subimage GL_KHR_debug GL_KHR_no_error GL_KHR_parallel_shader_compile GL_KHR_robust_buffer_access_behavior GL_NV_EGL_stream_consumer_external GL_NV_fence GL_NV_pack_subimage GL_NV_pixel_buffer_object GL_OES_EGL_image GL_OES_EGL_image_external GL_OES_EGL_image_external_essl3 GL_OES_compressed_EAC_R11_signed_texture GL_OES_compressed_EAC_R11_unsigned_texture GL_OES_compressed_EAC_RG11_signed_texture GL_OES_compressed_EAC_RG11_unsigned_texture GL_OES_compressed_ETC2_RGB8_texture GL_OES_compressed_ETC2_RGBA8_texture GL_OES_compressed_ETC2_punchthroughA_RGBA8_texture GL_OES_compressed_ETC2_punchthroughA_sRGB8_alpha_texture GL_OES_compressed_ETC2_sRGB8_alpha8_texture GL_OES_compressed_ETC2_sRGB8_texture GL_OES_depth24 GL_OES_depth32 GL_OES_draw_buffers_indexed GL_OES_draw_elements_base_vertex GL_OES_element_index_uint GL_OES_fbo_render_mipmap GL_OES_get_program_binary GL_OES_mapbuffer GL_OES_packed_depth_stencil GL_OES_rgb8_rgba8 GL_OES_standard_derivatives GL_OES_surfaceless_context GL_OES_texture_border_clamp GL_OES_texture_float GL_OES_texture_float_linear GL_OES_texture_half_float GL_OES_texture_half_float_linear GL_OES_texture_npot GL_OES_texture_stencil8 GL_OES_vertex_array_object GL_OVR_multiview GL_OVR_multiview2 GL_WEBGL_video_texture
WebGL 2 Extensions: EXT_color_buffer_float EXT_float_blend EXT_texture_compression_bptc EXT_texture_compression_rgtc EXT_texture_filter_anisotropic MOZ_debug OES_texture_float_linear OVR_multiview2 WEBGL_compressed_texture_s3tc WEBGL_compressed_texture_s3tc_srgb WEBGL_debug_renderer_info WEBGL_debug_shaders WEBGL_lose_context
Direct2D: true
Target Frame Rate: 60
DirectWrite: true (10.0.22000.258)
GPU #1
Active: Yes
Description: NVIDIA GeForce RTX 2060
Vendor ID: 0x10de
Device ID: 0x1e89
Driver Version: 30.0.15.1295
Driver Date: 5-19-2022
Drivers: C:\WINDOWS\System32\DriverStore\FileRepository\nv_dispig.inf_amd64_647b4244e991951b\nvldumdx.dll,C:\WINDOWS\System32\DriverStore\FileRepository\nv_dispig.inf_amd64_647b4244e991951b\nvldumdx.dll,C:\WINDOWS\System32\DriverStore\FileRepository\nv_dispig.inf_amd64_647b4244e991951b\nvldumdx.dll,C:\WINDOWS\System32\DriverStore\FileRepository\nv_dispig.inf_amd64_647b4244e991951b\nvldumdx.dll C:\WINDOWS\System32\DriverStore\FileRepository\nv_dispig.inf_amd64_647b4244e991951b\nvldumd.dll,C:\WINDOWS\System32\DriverStore\FileRepository\nv_dispig.inf_amd64_647b4244e991951b\nvldumd.dll,C:\WINDOWS\System32\DriverStore\FileRepository\nv_dispig.inf_amd64_647b4244e991951b\nvldumd.dll,C:\WINDOWS\System32\DriverStore\FileRepository\nv_dispig.inf_amd64_647b4244e991951b\nvldumd.dll
Subsys ID: 20683842
RAM: 6144
GPU #2
Active: No
RAM: 0
Diagnostics
AzureCanvasBackend: direct2d 1.1
AzureCanvasBackend (UI Process): skia
AzureContentBackend: skia
AzureContentBackend (UI Process): skia
AzureFallbackCanvasBackend (UI Process): skia
CMSOutputProfile: AAACKEtDTVMCAAAAbW50clJHQiBYWVogB9gACAAPAAgAAwA2YWNzcE1TRlQAAAAAAACsEAAAAAAAAAAAAAAAAAAAAAAAAPbVAAEAAAAA0ytLT0RBAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAJY3BydAAAAPAAAAAhd3RwdAAAARQAAAAUZGVzYwAAASgAAACqclhZWgAAAdQAAAAUZ1hZWgAAAegAAAAUYlhZWgAAAfwAAAAUclRSQwAAAhAAAAAWZ1RSQwAAAhAAAAAWYlRSQwAAAhAAAAAWdGV4dAAAAABDb3B5cmlnaHQgMjAwOCBEZWxsIEluYy4AAAAAWFlaIAAAAAAAAPbcAAEAAAAA0zpkZXNjAAAAAAAAABpEZWxsIDIyMDlXQQAAAENvbG9yUHJvZmlsZQAAAAAAAAAb/v8ARABlAGwAbAAgRS4yMjA5AFdBAAAAACAAQwBvAGwAbwByACAAUAByAG8AZgBpAGwAZQAAAAAaRGVsbEUyMjA5VwAAQ29sb3IgUHJvZmlsZQAyAP+Q8hIA//////z////lAwAAOAqVAOwz4XcA8hIAr/ISACDyEgBRdwAAWFlaIAAAAAAAAG4GAAA21AAAA79YWVogAAAAAAAAZYwAALbJAAAXHFhZWiAAAAAAAAAjQgAAEmEAALhQY3VydgAAAAAAAAAFAAAMIDe3h/L//wAA
Display0: 1050x1680@59Hz scales:1.000000|1.000000
Display1: 1680x1050@59Hz scales:1.000000|1.000000
DisplayCount: 2
HardwareStretching: both=1 window-only=0 full-screen-only=1 none=0 error=0
GPUProcessPid: 34584
GPUProcess: Terminate GPU Process
Device Reset: Trigger Device Reset
ClearType Parameters: \\.\DISPLAY1 [ Gamma: 1.8 Pixel Structure: RGB ClearType Level: 100 Enhanced Contrast: 50 ] \\.\DISPLAY2 [ Gamma: 1.8 Pixel Structure: RGB ClearType Level: 100 Enhanced Contrast: 50 ]
Decision Log
HW_COMPOSITING:
available by default
D3D11_COMPOSITING:
available by default
DIRECT2D:
available by default
D3D11_HW_ANGLE:
available by default
GPU_PROCESS:
available by default
WEBRENDER:
available by default
WEBRENDER_QUALIFIED:
available by default
WEBRENDER_COMPOSITOR:
available by default
WEBRENDER_PARTIAL:
available by default
WEBRENDER_SHADER_CACHE:
available by default
WEBRENDER_OPTIMIZED_SHADERS:
available by default
WEBRENDER_ANGLE:
available by default
WEBRENDER_DCOMP_PRESENT:
available by default
WEBRENDER_SOFTWARE:
available by default
WEBGPU:
disabled by default: Disabled by default
blocked by runtime: WebGPU cannot be enabled in release or beta
WINDOW_OCCLUSION:
available by default
VIDEO_OVERLAY:
blocked by default: Blocklisted by gfxInfo
HW_DECODED_VIDEO_ZERO_COPY:
available by default
VP8_HW_DECODE:
available by default
VP9_HW_DECODE:
available by default




Media
-----

Audio Backend: wasapi
Max Channels: 2
Preferred Sample Rate: 48000
Roundtrip latency (standard deviation): NaNms (NaN)
Output Devices
Name: Group
Speakers (Realtek(R) Audio): HDAUDIO\FUNC_01&VEN_10EC&DEV_0887&SUBSYS_104386C7&REV_1003\5&d2c1be6&0&0001
Headphones (Redmi Buds 3 Pro): BTHENUM\{0000110b-0000-1000-8000-00805f9b34fb}_VID&00010094_PID&0004\8&11ee0ad4&0&6CD3EE2DF49D_C00000000
Realtek Digital Output (Realtek(R) Audio): HDAUDIO\FUNC_01&VEN_10EC&DEV_0887&SUBSYS_104386C7&REV_1003\5&d2c1be6&0&0001
Headphones (JBL E25BT): BTHENUM\{0000110b-0000-1000-8000-00805f9b34fb}_VID&00010039_PID&1582\8&11ee0ad4&0&001441158AEC_C00000000
Input Devices
Name: Group
Headset (Redmi Buds 3 Pro): BTHHFENUM\BthHFPAudio\9&1454cee7&6&97
Microphone (Realtek(R) Audio): HDAUDIO\FUNC_01&VEN_10EC&DEV_0887&SUBSYS_104386C7&REV_1003\5&d2c1be6&0&0001
Stereo Mix (Realtek(R) Audio): HDAUDIO\FUNC_01&VEN_10EC&DEV_0887&SUBSYS_104386C7&REV_1003\5&d2c1be6&0&0001
Microphone (Logi Webcam C920e): USB\VID_046D&PID_08B6&MI_02\7&284398da&0&0002
Line In (Realtek(R) Audio): HDAUDIO\FUNC_01&VEN_10EC&DEV_0887&SUBSYS_104386C7&REV_1003\5&d2c1be6&0&0001

Enumerate database

Environment Variables
---------------------

MOZ_CRASHREPORTER_DATA_DIRECTORY: C:\Users\sobral\AppData\Roaming\Mozilla\Firefox\Crash Reports
MOZ_CRASHREPORTER_EVENTS_DIRECTORY: C:\Users\sobral\AppData\Roaming\Mozilla\Firefox\Crash Reports\events
MOZ_CRASHREPORTER_PING_DIRECTORY: C:\Users\sobral\AppData\Roaming\Mozilla\Firefox\Pending Pings
MOZ_CRASHREPORTER_RESTART_ARG_0: C:\Program Files\Firefox Developer Edition\firefox.exe
MOZ_CRASHREPORTER_STRINGS_OVERRIDE: C:\Program Files\Firefox Developer Edition\browser\crashreporter-override.ini

Experimental Features
---------------------

about:home startup cache (browser.startup.homepage.abouthome_cache.enabled): false
Cookies: SameSite=Lax by default (network.cookie.sameSite.laxByDefault): true
Cookies: SameSite=None requires secure attribute (network.cookie.sameSite.noneRequiresSecure): true
Cookies: Schemeful SameSite (network.cookie.sameSite.schemeful): true
CSS: Masonry Layout (layout.css.grid-template-masonry-value.enabled): false
Developer Tools: Compatibility Panel (devtools.inspector.compatibility.enabled): true
Developer Tools: Service Worker debugging (devtools.debugger.features.windowless-service-workers): false
Media: JPEG XL (image.jxl.enabled): true
Text Recognition (dom.text-recognition.enabled): false
Address Bar: show results during IME composition (browser.urlbar.keepPanelOpenDuringImeComposition): false
Web API: WebGPU (dom.webgpu.enabled): false
WebRTC Global Mute Toggles (privacy.webrtc.globalMuteToggles): false

Remote Experiments
------------------

TCP rollout, Aurora Phase III (TCP on by default for 100% Aurora profiles): control

Important Modified Preferences
------------------------------

accessibility.typeaheadfind.flashBar: 0
browser.contentblocking.category: custom
browser.download.useDownloadDir: false
browser.search.region: BR
browser.sessionstore.upgradeBackup.latestBuildID: 20220705185846
browser.startup.homepage_override.buildID: 20220705185846
browser.startup.homepage_override.mstone: 103.0
browser.startup.page: 3
browser.tabs.extraDragSpace: true
browser.urlbar.placeholderName: DuckDuckGo
browser.urlbar.placeholderName.private: DuckDuckGo
browser.urlbar.quicksuggest.migrationVersion: 2
browser.urlbar.quicksuggest.scenario: history
browser.urlbar.resultGroups: {"children":[{"maxResultCount":1,"children":[{"group":"heuristicTest"},{"group":"heuristicExtension"},{"group":"heuristi
browser.urlbar.tabToSearch.onboard.interactionsLeft: 0
browser.urlbar.tipShownCount.searchTip_onboard: 4
browser.urlbar.tipShownCount.searchTip_redirect: 1
doh-rollout.balrog-migration-done: true
doh-rollout.disable-heuristics: true
doh-rollout.doneFirstRun: true
doh-rollout.home-region: BR
extensions.lastAppVersion: 103.0
gfx.crash-guard.status.wmfvpxvideo: 2
gfx.crash-guard.wmfvpxvideo.appVersion: 97.0
gfx.crash-guard.wmfvpxvideo.deviceID: 0x1e89
gfx.crash-guard.wmfvpxvideo.driverVersion: 30.0.14.9729
idle.lastDailyNotification: 1657555917
image.jxl.enabled: true
media.gmp-gmpopenh264.abi: x86_64-msvc-x64
media.gmp-gmpopenh264.lastDownload: 1655485545
media.gmp-gmpopenh264.lastInstallStart: 1655485545
media.gmp-gmpopenh264.lastUpdate: 1655485546
media.gmp-gmpopenh264.version: 1.8.1.2
media.gmp-manager.buildID: 20220705185846
media.gmp-manager.lastCheck: 1657284533
media.gmp-manager.lastEmptyCheck: 1657284534
media.gmp-widevinecdm.abi: x86_64-msvc-x64
media.gmp-widevinecdm.lastUpdate: 1652883770
media.gmp-widevinecdm.version: 4.10.2449.0
media.gmp.storage.version.observed: 1
media.hardware-video-decoding.failed: false
media.videocontrols.picture-in-picture.video-toggle.has-used: true
network.dns.disablePrefetch: true
network.http.speculative-parallel-limit: 0
network.predictor.enabled: false
network.prefetch-next: false
network.trr.blocklist_cleanup_done: true
network.trr.mode: 2
network.trr.uri: https://mozilla.cloudflare-dns.com/dns-query
places.database.lastMaintenance: 1657290118
privacy.cpd.cache: false
privacy.cpd.cookies: false
privacy.cpd.downloads: false
privacy.cpd.history: false
privacy.cpd.sessions: false
privacy.purge_trackers.date_in_cookie_database: 0
privacy.purge_trackers.last_purge: 1657555918458
privacy.restrict3rdpartystorage.rollout.enabledByDefault: true
privacy.sanitize.pending: [{"id":"newtab-container","itemsToClear":[],"options":{}}]
privacy.trackingprotection.enabled: true
privacy.userContext.enabled: true
privacy.userContext.extension: {3c078156-979c-498b-8990-85f7987dd929}
privacy.userContext.ui.enabled: true
security.remote_settings.crlite_filters.checked: 1654024132
security.remote_settings.intermediates.checked: 1654002518
security.sandbox.content.tempDirSuffix: {4213e6ac-0f43-4bf0-a2bf-bd7d81b0b03d}
services.sync.declinedEngines: passwords
services.sync.engine.bookmarks.validation.lastTime: 1657490261
services.sync.engine.creditcards: true
services.sync.engine.extension-storage: true
services.sync.engine.forms: true
services.sync.engine.passwords: false
services.sync.engine.prefs.modified: false
services.sync.lastPing: 1657489656
services.sync.lastSync: Mon Jul 11 2022 15:46:04 GMT-0300 (Brasilia Standard Time)
signon.rememberSignons: false
storage.vacuum.last.index: 1
storage.vacuum.last.places.sqlite: 1656793816
ui.osk.debug.keyboardDisplayReason: IKPOS: Touch screen not found.

Important Locked Preferences
----------------------------

fission.autostart.session: true

Places Database
---------------

Accessibility
-------------

Activated: false
Prevent Accessibility: 0
Accessible Handler Used: true
Accessibility Instantiator:

Library Versions
----------------

NSPR
Expected minimum version: 4.34
Version in use: 4.34

NSS
Expected minimum version: 3.80
Version in use: 3.80

NSSSMIME
Expected minimum version: 3.80
Version in use: 3.80

NSSSSL
Expected minimum version: 3.80
Version in use: 3.80

NSSUTIL
Expected minimum version: 3.80
Version in use: 3.80

Sandbox
-------

Content Process Sandbox Level: 6
Effective Content Process Sandbox Level: 6
Win32k Lockdown State for Content Process: Win32k Lockdown enabled -- default value is true

Startup Cache
-------------

Disk Cache Path: C:\Users\sobral\AppData\Local\Mozilla\Firefox\Profiles\qc2l3gq3.dev-edition-default\startupCache\startupCache.8.little
Ignore Disk Cache: false
Found Disk Cache on Init: false
Wrote to Disk Cache: true

Internationalization & Localization
-----------------------------------

Application Settings
Requested Locales: ["en-US"]
Available Locales: ["en-US"]
App Locales: ["en-US"]
Regional Preferences: ["en-US"]
Default Locale: "en-US"
Operating System
System Locales: ["pt-BR"]
Regional Preferences: ["en-US"]

Remote Debugging (Chromium Protocol)
------------------------------------

Accepting Connections: false
URL:

Printing
--------

Modified print settings
-----------------------

print_printer: Microsoft Print to PDF
```

**Comment by dannycolin** [comment]
Duplicate of #1705
